Readme.rst

```diff
@@ -0,0 +1,25 @@
+This is a cookiecutter template for new Open edX plugins.
+
+It enables creation of the plugin with the best practices we at eduNEXT have compiled over time.
+
+We talk about it here:
+
+- https://youtu.be/0AdPHfjQ0wQ
+- https://docs.google.com/presentation/d/1ESyWbojf7tAtn4ODPTiUNOOh9JDuVmebdhpEBT5NSF8/edit?usp=sharing
+
+Usage
+=====
+
+To create a new plugin+ using this cookiecutter template, first make sure you have cookiecutter installed.
+
+Then run::
+
+        $ cookiecutter https://github.com/eduNEXT/cookiecutter-openedx-plugin.git
+
+Enter the variables when prompted.
+
+
+- project_desc -> human readable description of your plugin
+- package_name -> python package name. You can use _ for spaces
+- project_slug -> Normally also the repo name. You can use - or _ for spaces
+- class_name -> used when defining the app config python class
```

Comment:
Do you think we should add the list of files that the cookiecutter will create and his description? @felipemontoya

Reply:
Not really. I just listed this for people to have an idea on how to fill the variables. I would have preferred a variable description that comes in context but I did not see a way of doing that

Reply:
Ok.